RCS file: /home/kde/kdelibs/khtml/rendering/render_replaced.cpp,v
retrieving revision 1.92.2.5
retrieving revision 1.92.2.6
diff -b -p -u -r1.92.2.5 -r1.92.2.6
--- khtml/rendering/render_replaced.cpp	1 Oct 2002 19:22:51 -0000	1.92.2.5
+++ khtml/rendering/render_replaced.cpp	4 Jan 2003 23:42:31 -0000	1.92.2.6
@@ -153,7 +153,6 @@ void RenderWidget::detach()
 RenderWidget::~RenderWidget()
 {
     KHTMLAssert( refCount() <= 0 );
-    assert(!m_view);
 
     if(m_widget) {
         m_widget->hide();
